os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -477,61 +482,74 @@ void up_assert(const uint8_t *filename, int lineno)
 	}
 
 	irqstate_t flags = irqsave();
+
+#ifdef CONFIG_SECURITY_LEVEL
+	if (get_security_level() != LOW_SECURITY_LEVEL){
+		lldbg("*******************************************\n");
+		lldbg("Assertion failed\n");
+		lldbg("*******************************************\n");
+		lldbg("securit level: %d", get_security_level());
```

Comment:
Please fix the typo 
`securit` -> `security`

Reply:
Thank you~ I modified this